### PR TITLE
set different secret for app-secret from thoth token

### PR DESCRIPTION
**Pull request by harshad16**
set different secret for app-secret from thoth token
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

## Description

ensure separate secret for app-secret for the application and token for app-request-authentication

### CODE REVIEW

File: management-api/base/deploymentconfig.yaml [hunk 1]
```diff
@@ -88,11 +88,6 @@ spec:
                 configMapKeyRef:
                   key: bucket-prefix
                   name: ceph
-            - name: THOTH_SECRET
```
Comment (harshad16) [inline management-api/base/deploymentconfig.yaml:91]: i didnt see this being used, so removed it.

File: user-api/base/deploymentconfig.yaml [hunk 2]
```diff
@@ -41,7 +41,7 @@ spec:
             - name: THOTH_USER_API_TOKEN
               valueFrom:
                 secretKeyRef:
-                  key: thoth-secret
+                  key: user-api-token
```
Comment (harshad16) [inline user-api/base/deploymentconfig.yaml:44]: changed this key for consistency of component.

**Review by fridex — APPROVED** [review]
lgtm! 🎖️

File: management-api/base/deploymentconfig.yaml (hunk 1 above)
Comment (fridex) [reply management-api/base/deploymentconfig.yaml:91]: Verified, it is not used anywhere in sources 👍🏻

**Pull request merged**